HerPublicWebsite.BusinessLogic/Services/QuestionFlow/QuestionFlowService.cs

```diff
@@ -297,9 +301,13 @@ private QuestionFlowStep SelectAddressForwardDestination(Questionnaire questionn
 
         private QuestionFlowStep ReviewEpcForwardDestination(Questionnaire questionnaire, QuestionFlowStep? entryPoint)
         {
+            if (questionnaire.LocalAuthorityHug2Status is LocalAuthorityData.Hug2Status.Pending)
+            {
+                return QuestionFlowStep.Pending;
```

Comment:
⚠️ You need to make a corresponding change in the `PendingBackDestination` so that it can go back to the `ReviewEpc` page if it has come from there.

Reply:
I've made this change.